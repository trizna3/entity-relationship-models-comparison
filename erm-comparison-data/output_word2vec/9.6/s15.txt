persons-person
addresses-real estate
account-accounts
life insurance-life insurance
real estate insurance-real estate insurance
contract-contracts
real estate types-real estate types
EXTRACT_ATTR_TO_OWN_ENTITY_SET:S-person-life insurance
DECOMPOSE_ATTRIBUTE:S-address-real estate
