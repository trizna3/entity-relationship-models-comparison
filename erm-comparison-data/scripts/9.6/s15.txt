entity set accounts {
	"name"
	"password"
};

entity set contracts {
	"insurance type"
	"date from"
	"date to"
	"fee"
};

entity set "life insurance" {
	"person"
	"injury amount"
	"work incapacity"
	"permanent consequences"
	"death insurance"
};

entity set "person" {
	"data"
};

entity set "real estate insurance" {
	"real estate"
	"crashes"
	"elemental disaster"
};

entity set "real estate" {
	"address"
	"type"
	"living area size"
	"basement size"
};

entity set "real estate types" {
	"flat"
	"house"
	"office"
};

association between 
{
	accounts ONE 
	contracts MANY
}{};

association between 
{
	contracts ONE 
	"life insurance" ONE
}{};

association between 
{
	contracts ONE 
	"real estate insurance" ONE
}{};

association between 
{
	"life insurance" MANY 
	"person" ONE
}{};

association between 
{
	"real estate insurance" MANY 
	"real estate" ONE
}{};

association between 
{
	"real estate" ONE 
	"real estate types" ONE
}{};