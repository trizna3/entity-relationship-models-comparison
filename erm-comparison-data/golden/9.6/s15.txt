account-accounts
contract-contracts
life insurance-life insurance
real estate insurance-real estate insurance
persons-person
addresses-address
real estate types-real estate types
EXTRACT_ATTR_TO_OWN_ENTITY_SET:S-address-real estate